src/Examples/Flowsheets/methanol_synthesis.ipynb

```diff
@@ -0,0 +1,1749 @@
+      "    Total Mole Fraction CH4    dimensionless 4.4069e-05\n",
+      "    Total Mole Fraction CO     dimensionless    0.25377\n",
+      "    Total Mole Fraction H2     dimensionless    0.67379\n",
+      "    Total Mole Fraction CH3OH  dimensionless   0.072392\n",
+      "    Molar Enthalpy              joule / mole    -39572.\n",
+      "    Pressure                          pascal 1.2103e+07\n",
+      "====================================================================================\n"
+     ]
+    }
+   ],
+   "source": [
+    "# Set up Optimization Problem (Maximize Revenue)\n",
+    "# keep process pre-reaction fixed and unfix some post-process specs\n",
+    "m.fs.R101.conversion.unfix()\n",
+    "m.fs.R101.conversion_lb = Constraint(expr=m.fs.R101.conversion >= 0.75)\n",
+    "m.fs.R101.conversion_ub = Constraint(expr=m.fs.R101.conversion <= 0.85)\n",
+    "m.fs.R101.outlet_temp.deactivate()\n",
+    "m.fs.R101.outlet_t_lb = Constraint(\n",
+    "    expr=m.fs.R101.control_volume.properties_out[0.0].temperature >= 405)\n",
+    "m.fs.R101.outlet_t_ub = Constraint(\n",
+    "    expr=m.fs.R101.control_volume.properties_out[0.0].temperature <= 505)\n",
+    "\n",
+    "# Optimize turbine work (or delta P)\n",
+    "m.fs.T101.deltaP.unfix()  # optimize turbine work recovery/pressure drop\n",
+    "m.fs.T101.outlet_p_lb = Constraint(\n",
+    "    expr=m.fs.T101.outlet.pressure[0] >= 10E5)\n",
+    "m.fs.T101.outlet_p_ub = Constraint(\n",
+    "    expr=m.fs.T101.outlet.pressure[0] <= 51E5*0.8)\n",
+    "\n",
+    "# Optimize Cooler outlet temperature - unfix cooler outlet temperature\n",
+    "m.fs.H102.outlet_temp.deactivate()\n",
+    "m.fs.H102.outlet_t_lb = Constraint(\n",
+    "    expr=m.fs.H102.control_volume.properties_out[0.0].temperature\n",
+    "    >= 407.15*0.8)\n",
+    "m.fs.H102.outlet_t_ub = Constraint(\n",
+    "    expr=m.fs.H102.control_volume.properties_out[0.0].temperature\n",
+    "    <= 480)\n",
+    "\n",
+    "m.fs.F101.deltaP.unfix()  # allow pressure change in streams\n",
+    "\n",
+    "m.fs.F101.isothermal = Constraint(\n",
+    "    expr=m.fs.F101.control_volume.properties_out[0].temperature ==\n",
+    "    m.fs.F101.control_volume.properties_in[0].temperature)\n",
+    "\n",
+    "m.fs.S101.split_fraction[0, \"purge\"].unfix()  # allow some gas recycle\n",
+    "m.fs.S101.split_fraction_lb = Constraint(\n",
+    "    expr=m.fs.S101.split_fraction[0, \"purge\"] >= 0.10)  # min 10% purge\n",
+    "m.fs.S101.split_fraction_ub = Constraint(\n",
+    "    expr=m.fs.S101.split_fraction[0, \"purge\"] <= 0.50)  # max 50% purge\n",
+    "\n",
+    "print()\n",
+    "print('Solving optimization problem...')\n",
+    "opt_res = solver.solve(m, tee=True)\n",
+    "assert opt_res.solver.termination_condition == TerminationCondition.optimal\n",
+    "print('Optimal solution process results:')\n",
+    "report(m)"
+   ]
+  },
+  {
+   "cell_type": "code",
```

Comment:
It is hard to tell via the GitHub interface, but this appears to be missing the tags to indicate that this is a testing cell and should not be included in the public facing version,